generators/csharp/base/src/asIs/RawClient.Template.cs

```diff
@@ -214,7 +214,7 @@ and not string
                 {
                     var items = collection
                         .Cast<object>()
-                        .Select(value => $"{Uri.EscapeDataString(queryItem.Key)}={Uri.EscapeDataString(value.ToString())}")
+                        .Select(value => $"{Uri.EscapeDataString(queryItem.Key)}={Uri.EscapeDataString(value.ToString() ?? "null")}")
```

Comment:
I vaguely remember there are some classes that handle conversion to query string in the C# generator.

Reply:
The `ValueConvert` class with `ToQueryStringValue`. It also converts `null` to `""`, not `"null"`. Is there a customer that is asking for it to convert to `"null"`?

Reply:
No. And now that I'm looking at this, I have no idea why I put `"null"` in there.  Fixing.